add content for module 1 (#2)
microsoft/aed-rust-learn-content: take-your-first-steps-with-rust/get-started-with-rust/includes/2-install.md

# Install Rust

The recommended way to install Rust is to use `rustup`, the Rust Toolchain Installer. Go to

> <https://rustup.rs> to find the appropriate instructions for your operating system.

Reviewer: ```suggestion
Go to the website [rustup.rs](https://rustup.rs) to find the appropriate instructions for your operating system.
```

> 

Reviewer: Could you add a screenshot at this point? I attached one here for reference that you can use.

![rustup](https://user-images.githubusercontent.com/10103121/98585671-7dfdb480-227c-11eb-9078-7a303fe0da07.png)

Note that screenshots typically need to include the full window of the browser or whatever is being captured.

Then you can put the image in the `media/` folder and reference it in this file like:
:::image type="content" source="../media/rustup.png" alt-text="Image of rustup installer website.":::

Reviewer: Could you add a bit more direction for the installation?  For example:
```suggestion
Copy the curl command by selecting the clipboard icon and then open up your computer's Terminal or Command Prompt to paste the command.

Wait a few minutes for the installation to complete.
```

Rust has a **6-week rapid release process** and supports a great number of platforms, so there are many
builds of Rust available at any time. So, if you've installed `rustup` in the past, you can update
your installation by running the command `rustup update`.

> After you’ve completed the installation, you should have the `rustc` and `cargo` commands available at your command line:

Reviewer: ```suggestion
After you’ve completed the installation, you should have the `rustc` and `cargo` commands available. Type the following commands in your Terminal or Command Prompt:
```
